src/util/duration.h

```diff
@@ -136,19 +157,19 @@ class Duration {
     }
 
     QString formatNanosWithUnit() const {
-        return QString("%1ns").arg(toNanos());
+        return QString("%1ns").arg(toIntegerNanos());
```

Comment:
"50 s music"  = a music with a duration of 50 s
"50s music" = music released in years 1950 to 1959